tests/multisig.ts

```diff
@@ -1147,6 +1163,154 @@ describe("multisig", () => {
       });
     });
   });
+
+  describe("end-to-end scenarios", () => {
+    it('transaction with additional "ephemeral" signers', async () => {
```

Comment:
This test illustrates the case where an ephemeral keypair is required to sign the underlying transaction.